corehq/apps/reports/standard/cases/basic.py

```diff
@@ -328,6 +328,13 @@ def get_data(self):
 
             yield data
 
+    @property
+    def template_context(self):
+        context = {
+            'user_types': HQUserType.human_readable,
+        }
+        return context
```

Comment:
nit: why use four lines when it would all fit comfortably on one?
```py
return {'user_types': HQUserType.human_readable}
```

Reply:
Good point: https://github.com/dimagi/commcare-hq/pull/22079/commits/3401d397d18af1ff745b72f2f953303068250aca